### ISSUE TITLE
HTML Font tag 

### ISSUE THREAD

**Issue by saudansari** [xdan/jodit#206]
**Jodit Version:**  3.2.44

**Code**
No Code Change

**Expected behavior:**
When I change the font size, the font size should change. HTML shld reflect font-size style attribute is used. 
**Actual behavior:**
When i change the font size to any size, it changes to size <font size="7"> using the Font tag which is not supported by HTML5. style fount-size does not get used. 
I have no idea where <font> is coming from. 

Updated 28-05-2019
worth mentioning, Yes, running it in prod build i get the issue. dev build works fine.

**Comment by masbaehr** [comment]
I also noticed this, it seems to happen only when i build the project. in Dev mode (npm) this is not happening

**Comment by anthonymartinovic** [comment]
I'm experiencing this same issue too. I haven't found a solution to this yet, but I was able to recreate the problem in Jodit playground by simply disabling the font plugin. It only happens in a prod build for me too.

**Comment by amrita-syn** [comment]
Same issue - https://github.com/xdan/jodit/issues/225

@xdan has marked it as "help-wanted". If font-size not being supported in html5 were a problem, then maybe this could be the direction we need to look at?

**Comment by xdan** [comment]
@saudansari  Which browser is the problem? I tried to repeat here https://xdsoft.net/jodit/ and everything works as it should

**Comment by saudansari** [comment]
@xdan i tried it on Chrome (latest version). As @masbaehr mentioned, in dev mode it works fine, but when we deploy it, we start getting the issue. If there is any specific config we need to add to the config, please let us know.

**Comment by anthonymartinovic** [comment]
@saudansari @xdan This problem occurs because of Angular's optimization process. I assume most other frameworks would encounter similar issues too.

Angular has its own locked-down configuration for optimization (I believe they use UglifyJS too), but it's configuration will break Jodit.

If you view Jodit's optimization config in the webpack config file you'll see some properties, such as 'Jodit' being a reserved keyword, are required for Jodit to work correctly. Angular doesn't provide any access to these config options so it'll break when your app is minified.

I found most of the Jodit plugins didn't work because of this problem. For the time being in my project I've just added Jodit's minified code as a script after my app has finished its optimization process, but it's not a good solution.

**Comment by amrita-syn** [comment]
@anthonymartinovic thanks for your input. 

@xdan, I had tried adding the jodit min as script in react app but couldn't access it correctly in the component. I don't know if it will solve the issue for me but can you suggest how to implement the solution suggested by anthonymartinovic in the react setup?

**Comment by amrita-syn** [comment]
> If you view Jodit's optimization config in the webpack config file you'll see some properties, such as 'Jodit' being a reserved keyword, are required for Jodit to work correctly. Angular doesn't provide any access to these config options so it'll break when your app is minified.

@anthonymartinovic I thought after reading your reply that why not remove the uglify and minify step. But looking at webpack config and package json, I see that the 'production-no-uflify' flag is set and the uglify and minify steps are not happening any way. Any other ideas?

**Comment by tigrankeshishyan** [comment]
Hello guys.
Did someone found a solution for this issue (even by hacking or hardcoding)?

**Comment by amrita-syn** [comment]
@tigrankeshishyan  I did. Add it as a script tag in `public/index.html`. 
```
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/jodit/3.2.55/jodit.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/jodit/3.2.55/jodit.min.js"></script>
```
And then use in components like so -
```
const Jodit = window.Jodit;
const editor = new Jodit('#editor');
```

**Comment by tigrankeshishyan** [comment]
@amrita-syn  Thank you for your answer but that didn't help.
In production mode is still uses <font> tag to wrap the selected element, but in dev mode it wraps with <span> and adds appropriate font-size to styles

**Comment by tigrankeshishyan** [comment]
anyway, guys, I found a hack solution for the situation and it works in dev and in prod as well.

Let's say we have these font sizes generated.

```
const generatedFontSizes = {};

for (let i = 8; i <= 30; i++) {
  generatedFontSizes[i] = `${i} size`;
}
```
Then we can remove (or keep) the fontSize button from ttolbar and add our icon with new functionality.

`const defaultJoditBtns = xor(Jodit.defaultOptions.buttons, ['fontsize']);`

```
const joditOptions = {
   buttons: defaultJoditBtns.concat([{
    name: 'Font Size',
    icon: 'fontsize',
    list: generatedFontSizes,
    exec: (editor, text, controls) => {
      editor.selection.applyCSS({
        'font-size': `${Number(controls.args[0])}px`,
      });
    },
    template: (key, value) => '<div>' + value + '</div>',
  },
  ]),
};
```
Hope to help someone else. 
Thanks.

**Comment by corrrso** [comment]
Exact same problem here. this should be fixed ASAP.

**Comment by corrrso** [comment]
This seems to be solved in version 3.3.22 https://github.com/xdan/jodit/releases/tag/3.3.22
